application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -32,24 +35,33 @@ public TagCommand(TagSystem tagSystem) {
 
         this.tagSystem = tagSystem;
 
-        // TODO Thing about adding an ephemeral selection menu with pagination support
+        // TODO Think about adding an ephemeral selection menu with pagination support
         // if the user calls this without id or similar
-        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true);
+        getData().addOption(OptionType.STRING, ID_OPTION, "the id of the tag to display", true)
+            .addOption(OptionType.USER, MENTION_USER_OPTION,
+                    "optionally, the user who you want to reply to", false);
     }
 
     @Override
     public void onSlashCommand(@NotNull SlashCommandEvent event) {
         String id = Objects.requireNonNull(event.getOption(ID_OPTION)).getAsString();
+        OptionMapping mentionUserOption = event.getOption(MENTION_USER_OPTION);
```

Comment:
If u rename the constant, then you might also want to rename the variable here to `replyToUserOption`